docs-source/docs/modules/how-to/examples/shopping-cart-service-cassandra-java/pom.xml

```diff
@@ -75,8 +76,15 @@
         <dependency>
             <groupId>org.apache.pekko</groupId>
             <artifactId>pekko-discovery_${scala.binary.version}</artifactId>
-            <version>${pekko.version}</version>
+            <version>${pekko.discovery.version}</version>
```

Comment:
this is wrong - pekko-discovery is part of the core pekko and should use v1.0.2

pekko-discovery-kubernetes-api is part of pekko-management and should use pekko-management.version